src/renderer/shared/ui/EmptyList/EmptyList.tsx

```diff
@@ -1,18 +1,23 @@
 import { type PropsWithChildren, type ReactNode } from 'react';
 
 import { Graphics } from '@/shared/ui-kit';
-import { BodyText } from '../Typography';
+import { BodyText, SmallTitleText } from '../Typography';
 
 type Props = {
   iconAlt?: string;
+  title?: ReactNode;
   message: ReactNode;
 };
 
-export const EmptyList = ({ message, iconAlt = '', children }: PropsWithChildren<Props>) => {
+export const EmptyList = ({ title, message, iconAlt = '', children }: PropsWithChildren<Props>) => {
   return (
     <div className="flex h-full flex-col items-center justify-center gap-4">
       <Graphics name="emptyList" alt={iconAlt} size={178} />
-      <BodyText className="w-[300px] text-center text-text-tertiary">{message}</BodyText>
+
+      <div className="flex flex-col items-center gap-3">
+        {title && <SmallTitleText>{title}</SmallTitleText>}
```

Comment:
`TextBase` has this condition inside `if (!children) return null;`

```suggestion
        <SmallTitleText>{title}</SmallTitleText>
```